src/transformers/integrations.py

```diff
@@ -174,16 +173,17 @@ def _objective(trial, local_trainer, checkpoint_dir=None):
 
     _tb_writer = trainer.pop_callback(TensorBoardCallback)
     trainer.model = None
-    # Setup default `resources_per_trial` and `reporter`.
-    if "resources_per_trial" not in kwargs and trainer.args.n_gpu > 0:
-        # `args.n_gpu` is considered the total number of GPUs that will be split
-        # among the `n_jobs`
-        n_jobs = int(kwargs.pop("n_jobs", 1))
```

Comment:
you should also output that n_jobs is no longer supported?

Reply:
Good point I’ll add a log statement here saying what resources_per_trial is being set to if it’s not being passed in by the user.

I actually don’t think n_jobs should have been supported here in the first place- that’s an optuna specific arg.

Reply:
Log statement added